### PR TITLE
Update Safari versions for Geolocation APIs

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
This PR updates and corrects the real values for Safari (Desktop and iOS/iPadOS) for the `GeolocationCoordinates` API, based upon results from the [mdn-bcd-collector](https://mdn-bcd-collector.appspot.com) project (v3.1.4).

Tests Used:
https://mdn-bcd-collector.appspot.com/tests/api/Geolocation
https://mdn-bcd-collector.appspot.com/tests/api/GeolocationCoordinates
https://mdn-bcd-collector.appspot.com/tests/api/GeolocationPosition

### CODE REVIEW

File: api/GeolocationCoordinates.json [hunk 1]
```diff
@@ -72,14 +72,24 @@
...
-            "alternative_name": "Coordinates",
-            "version_added": "4.2"
-          },
+          "safari": [
+            {
+              "version_added": "13.1"
+            },
+            {
+              "alternative_name": "Coordinates",
+              "version_added": "5"
```
Comment (foolip) [inline api/GeolocationCoordinates.json:81]: This matches api.GeolocationPosition.coords, good!

But should there also be a `version_removed`?

File: api/GeolocationCoordinates.json [hunk 2]
```diff
@@ -146,7 +156,7 @@
               "version_added": "5"
             },
             "safari_ios": {
-              "version_added": null
+              "version_added": "5"
```
Comment (foolip) [inline api/GeolocationCoordinates.json:159]: The above edit says 4.2, and api.Navigator.geolocation does too. But https://caniuse.com/geolocation suggests this was shipped on iOS before macOS, which actually makes a lot of sense for this API. Perhaps leave the iOS data alone, and do a separate PR to make it consistent?

**Review by foolip — CHANGES_REQUESTED** [review]

File: api/GeolocationCoordinates.json (hunk 1 above)
Comment (vinyldarkscratch) [reply api/GeolocationCoordinates.json]: Added based upon testing the alt. name's existence in Safari 13.1!

File: api/GeolocationCoordinates.json (hunk 2 above)
Comment (foolip) [reply api/GeolocationCoordinates.json:161]: For the avoidance of doubt: I'm waiting for this before reviewing again.

Comment (vinyldarkscratch) [reply api/GeolocationCoordinates.json:161]: It does seem like support on iOS has been around a lot longer per testing in BrowserStack on iOS 3.  I'll set this to `≤3` for iOS accordingly, and update the other Geolocation APIs as well.

Comment (foolip) [reply api/GeolocationCoordinates.json]: Wohoo!

**Review by foolip — APPROVED** [review]
